### PR TITLE
SUBMARINE-482. Implement database connection in team page with Angular

### PR DESCRIPTION
**Pull request by kobe860219**
### What is this PR for?
Database connection in team page with Angular.
Functions about :
1.List exist teams.
2.Add new team.
3.Delete an exist team.
4.Easy check when add new team.
Above is done.
Something may need to dicuss after version 0.4 released.

Future - Owner value may need to auto-catch user.

### What type of PR is it?
[Improvement]

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-482

### How should this be tested?
https://travis-ci.org/github/kobe860219/submarine/builds/685974451

### Screenshots (if appropriate)
TestIT
teamIT test

me80l-utn3q

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by kobe860219** [comment]
@liuxunorg  @pingsutw  @jasoonn  Please help me review this pr. Thanks! And I have an error about check license , I am not sure where problem is.

**Comment by pingsutw** [comment]
There is a `mysql` file with no content, delete it will fix the test

File: submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/teamIT.java [hunk 1]
```diff
@@ -60,12 +60,12 @@ public void teamTest() throws Exception {
...
-    pollingWait(By.xpath("//textarea[@id='textareaNewTeamDescription']"), MAX_BROWSER_TIMEOUT_SEC).sendKeys("NewTeamTest");
-    clickAndWait(By.cssSelector("button[id='summitCreateTeam']"));
-    Assert.assertEquals(pollingWait(By.xpath("//button[@id='createTeamBtn']"), MAX_BROWSER_TIMEOUT_SEC).isDisplayed(), true);
-
+    clickAndWait(By.cssSelector("button[id='btnAddTeam']"));
+    pollingWait(By.xpath("//input[@id='inputNewTeamName']"), MAX_BROWSER_TIMEOUT_SEC).sendKeys("NewTeamNameTest");
+    pollingWait(By.xpath("//input[@id='inputNewTeamOwner']"), MAX_BROWSER_TIMEOUT_SEC).sendKeys("NewTeamOwnerTest");
+    clickAndWait(By.cssSelector("button[id='submitNewTeamBtn']"));
+    Assert.assertEquals(pollingWait(By.xpath("//td[contains(., 'NewTeamNameTest')]"), MAX_BROWSER_TIMEOUT_SEC).isDisplayed(), true);
+
```
Comment (pingsutw) [inline submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/teamIT.java:68]: Should remove redundant whitespaces.
overall LGTM.

Comment (kobe860219) [reply submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/teamIT.java]: Has done!

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/workspace/team/team.component.html [hunk 2]
```diff
@@ -17,176 +17,94 @@
...
+<nz-modal [nzVisible]="overviewModalVisible" [nzWidth]="540" nzTitle="Overview" (nzOnCancel)="handleCancel()" (nzOnOk)="handleOk()">
+  <h2>Team Name:</h2>
+  <p>{{ currentTeam_teamName }}</p>
+  <h2>ID:</h2>
+  <p>{{  currentTeam_id  }}</p>
+  <h2>Owner:</h2>
+  <p>{{  currentTeam_owner  }}</p>
+  <h2>Create Time</h2>
+  <p>{{  currentTeam_createTime  }}</p>
+</nz-modal>
```
Comment (jasoonn) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/workspace/team/team.component.html:110]: No newline at the end of the file.

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/workspace/team/team.component.scss [hunk 3]
```diff
@@ -17,12 +17,23 @@
...
+ .footer{
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    border-top: 1px solid rgb(232, 232, 232);
+    padding: 10px 16px;
+    text-align: right;
+    left: 0px;
+    background: #fff;
  }
```
Comment (jasoonn) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/workspace/team/team.component.scss:39]: No newline at the end of the file.

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/workspace/team/team.component.html (hunk 2 above)
Comment (kobe860219) [reply submarine-workbench/workbench-web-ng/src/app/pages/workbench/workspace/team/team.component.html]: Added done.

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/workspace/team/team.component.scss (hunk 3 above)
Comment (kobe860219) [reply submarine-workbench/workbench-web-ng/src/app/pages/workbench/workspace/team/team.component.scss:39]: Added done

**Comment by liuxunorg** [comment]
@kobe860219 Thank you contribtion this feature.
Can you attach screenshots of several workbench team page operations.

**Comment by kobe860219** [comment]
@liuxunorg Hi, I have updated a .gif for showing current team page. If need more, please tell me.I will start to study about team member function  . More details about team page may need discuss after version 0.4 released.

**Pull request closed without merging**